app/models/storage.rb

```diff
@@ -70,6 +71,12 @@ class Storage < ApplicationRecord
 
   SUPPORTED_STORAGE_TYPES = %w( VMFS NFS FCP ISCSI GLUSTERFS )
 
+  supports :smartstate_analysis do
+    if ext_management_systems.blank? || !ext_management_system.kind_of?(ManageIQ::Providers::Vmware::InfraManager)
```

Comment:
@roliveri this was the discussion on gitter [July 21, 2016 4:10 PM](https://gitter.im/ManageIQ/manageiq/providers?at=5790d7d29f35137e67f802cc)